WindowManager/app/src/main/java/com/example/windowmanagersample/embedding/SplitActivityTrampoline.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.windowmanagersample.embedding
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.LayoutDirection
+import androidx.window.core.ExperimentalWindowApi
+import androidx.window.embedding.ActivityFilter
+import androidx.window.embedding.SplitController
+import androidx.window.embedding.SplitPlaceholderRule
+
+/**
+ * Example trampoline activity that launches a split and finishes itself.
+ */
+@ExperimentalWindowApi
+class SplitActivityTrampoline : SplitActivityBase() {
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
When launching `SplitActivityTrampoline` via the launcher, I'm getting a crash right now of a `android.content.ActivityNotFoundException`.

Reply:
I had some references to the `androidx.window.sample` package left over, it should be fine now.

Reply:
Looks good now!